tests/org.eclipse.n4js.flowgraphs.tests/model/DeadCodeAnalysis.n4js.xt

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
test files that are more about a validation than the internals of the flow graph engine(s) should better go into the spec-tests bundle

Reply:
done